docs/bigquery-gettingstarted.md

```diff
@@ -0,0 +1,150 @@
+# Getting Started Accessing the HTTP Archive with BigQuery
+
+The HTTP Archive is open source and the [raw data is available to download](http://httparchive.org/downloads.php).   If you want, you can download all of the data, process it locally and query it as much as you want.   However, that would require alot of effort to set up.   Google BigQuery makes this much easier, since the data is already loaded and can be easily queried. And because of the processing power behind BigQuery, even some of the most complex queries you write will run extremely fast. 
+
+This document is an update to [Ilya Grigorik's 2013 introduction](https://www.igvita.com/2013/06/20/http-archive-bigquery-web-performance-answers/), and will walk you through everything you need to get started accessing BigQuery and analyzing the data.
+
+## Setting up BigQuery to Access the HTTP Archive
+
+In order to access the HTTP Archive via BigQuery, you'll need a Google account.  To document this process for new visitors, this example uses a new Google account that has never logged into any Google Cloud services.
+
+1. Navigate to the BigQuery QuickStart page - https://cloud.google.com/bigquery/quickstart-web-ui
+2. Click on the [Sign In](https://accounts.google.com/Login) link to log into your Google account.
+3. Once logged in, go to the [Projects Page](https://console.cloud.google.com/start).  If this is your first time accessing Google Cloud, you may be prompted to accept the ToS.
```

Comment:
Are the previous two steps necessary? If they start here, won't it prompt them to sign in?

Reply:
You're right.   Tested it to confirm.   Will remove the first 2 steps.